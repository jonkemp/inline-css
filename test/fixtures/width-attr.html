<html>
<head>
    <style>
        p, td, img {
            width: 200px;
        }
    </style>
</head>
<body>
    <p>none</p>
    <table>
        <tr>
            <td>
                wide
            </td>
        </tr>
    </table>
    <img src="" alt="wide"></body></html>
