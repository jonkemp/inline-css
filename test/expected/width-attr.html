<html>
<head>
    
</head>
<body>
    <p style="width: 200px;">none</p>
    <table>
        <tbody>
            <tr>
                <td style="width: 200px;" width="200">
                    wide
                </td>
            </tr>
        </tbody>
    </table>
    <img src="" alt="wide" style="width: 200px;" width="200">
</body>
</html>
